pacoloco.go

```diff
@@ -293,12 +319,18 @@ func handleRequest(w http.ResponseWriter, req *http.Request) error {
 		for _, url := range repo.getUrls() {
 			downloaded, err = downloadFile(url+path+"/"+fileName, filePath, ifLater, w)
 			if err == nil {
+				if downloaded {
+					downloadedFiles.WithLabelValues(repoName).Inc()
+				}
 				break
 			}
+			abortedDownloads.WithLabelValues(repoName).Inc()
 		}
 	}
-	if !downloaded {
+
+	if !downloaded && err == nil {
```

Comment:
If none of the upsteams managed to download files, it would still trip cache - despite no file being cached.

Reply:
The database files (*.db) is an example when we want to check/download upstream, but even if the operation fails we still want to serve it from the local cache.

Reply:
Haven't thought of that use-case. I rewrote that to check if the file is present and can be served. This matches the current behavior with added log entry and counter.